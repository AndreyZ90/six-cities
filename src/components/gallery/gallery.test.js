import React from 'react';
import TestRenderer from 'react-test-renderer';

import Gallery from '@/components/gallery/gallery';

const images = [`img/room.jpg`, `img/hotel.jpg`];

describe(`Gallery component snapshot`, () => {
  test(`Should correctly render Gallery component`, () => {
    const tree = TestRenderer.create(<Gallery images={images} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
