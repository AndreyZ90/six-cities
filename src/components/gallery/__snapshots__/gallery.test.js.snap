// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component snapshot Should correctly render Gallery component 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="/img/room.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="/img/hotel.jpg"
      />
    </div>
  </div>
</div>
`;
